### PR TITLE
chore: add github actions dependabot config

### PR DESCRIPTION
**Pull request by mmourick**
## What
Adds Github actions dependabot config.

## Why
Automating github actions maintenance reduces manual toil.

## References
N/A

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (bschaatsbergen) [inline .github/dependabot.yaml:10]: Considering we have a daily cron for gomod, should we run daily for github-actions too?

**Review by bschaatsbergen — APPROVED** [review]
LGTM :rock

File: .github/dependabot.yaml (hunk 1 above)
Comment (mmourick) [reply .github/dependabot.yaml]: Sounds good to me! Updated in 2761ecdee07cd8bc7ce816b5902d150938140906

**Review by bschaatsbergen — APPROVED** [review]
LGMT 🚀

**Pull request merged**